experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -124,11 +121,12 @@ class ExPlat internal constructor(
             experiments: Set<Experiment>,
             logger: ExperimentLogger,
             coroutineScope: CoroutineScope,
+            dispatcher: CoroutineDispatcher = Dispatchers.IO,
```

Comment:
Suggestion (🔍): I suggest moving `dispatcher` and `coroutineScope` at the end of the function's signatures, them becoming the last parameters in it.

Reply:
Added to: https://github.com/Automattic/Automattic-Tracks-Android/pull/243#discussion_r1738128236